editor/SCsub

```diff
@@ -41,7 +41,7 @@ def make_doc_header(target, source, env):
         src = s.srcnode().abspath
         f = open_utf8(src, "r")
         content = f.read()
-	buf+=content
+        buf+=content
```

Comment:
Please use spaces around operators.

Reply:
Or nevermind, there are several occurrences that need to be fixed with autopep8 anyway, I'll have a go at it later.